bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.humidifier.ca4.json

```diff
@@ -0,0 +1,271 @@
+				"type": "Number",
+				"refresh": true,
+				"ChannelGroup": "Status",
+				"actions": [
+					{
+						"command": "set_properties",
+						"parameterType": "NUMBER"
+					}
+				]
+			},
+			{
+				"property": "water_level",
+				"siid": 2,
+				"piid": 7,
+				"friendlyName": "Water Level",
+				"channel": "waterlevel",
+				"channelType": "waterlevel",
+				"type": "Number",
+				"refresh": true,
+				"ChannelGroup": "Status",
+                "transformation": "tankLevel",
+				"actions": []
+			},
+			{
+				"property": "led_brightness",
+				"siid": 5,
+				"piid": 2,
+				"friendlyName": "LED Brightness",
+				"channel": "bright",
+				"channelType": "humidifierLedBrightness",
+				"type": "Number",
+				"refresh": true,
+				"ChannelGroup": "actions",
+				"actions": [
+					{
+						"command": "set_properties",
+						"parameterType": "NUMBER"
+					}
+				]
+			},
+			{
+				"property": "buzzer",
+				"siid": 4,
+				"piid": 1,
+				"friendlyName": "Buzzer Status",
+				"channel": "buzzer",
+				"channelType": "buzzer",
+				"type": "Switch",
+				"refresh": true,
+				"ChannelGroup": "actions",
+				"actions": [
+					{
+						"command": "set_properties",
+						"parameterType": "ONOFFBOOL"
+					}
+				]
+			},
+			{
+				"property": "dry",
+				"siid": 2,
+				"piid": 8,
+				"friendlyName": "Dry",
+				"channel": "dry",
+				"channelType": "dry",
+				"type": "Switch",
+				"refresh": true,
+				"ChannelGroup": "Status",
+				"actions": [
+					{
+						"command": "set_properties",
+						"parameterType": "ONOFFBOOL"
+					}
+				]
+			},
+			{
+				"property": "use_time",
+				"siid": 2,
+				"piid": 9,
+				"friendlyName": "Run Time",
+				"channel": "usedhours",
+				"channelType": "usedhours",
+				"type": "Number",
+				"refresh": true,
+				"transformation": "SecondsToHours",
+				"ChannelGroup": "Status",
+				"actions": []
+			},
+			{
+				"property": "power_time",
+				"siid": 7,
+				"piid": 3,
+				"friendlyName": "Power Time",
+				"channel": "powerhours",
+				"channelType": "powerhours",
+				"type": "Number",
+				"refresh": true,
+				"transformation": "SecondsToHours",
+				"ChannelGroup": "Status",
+				"actions": []
+			},
+			{
+				"property": "speed_level",
+				"siid": 2,
+				"piid": 11,
+				"friendlyName": "Target Motor Speed",
+				"channel": "targetmotorspeed",
+				"channelType": "motorspeed",
+				"type": "Number",
+				"refresh": true,
+				"ChannelGroup": "Status",
+				"actions": []
+			},
+			{
+				"property": "actual_speed",
+				"siid": 7,
+				"piid": 1,
+				"friendlyName": "Actual Motor Speed",
+				"channel": "actualmotorspeed",
+				"channelType": "motorspeed",
+				"type": "Number",
+				"refresh": true,
+				"ChannelGroup": "Status",
+				"actions": []
+			},
+			{
+				"property": "temp_dec",
+				"siid": 3,
+				"piid": 7,
+				"friendlyName": "Temperature",
+				"channel": "temperature",
+				"channelType": "temperatureCelsius",
+				"type": "Number",
```

Comment:
Number:Temperature
can re-use Channeltype temperatureC unless you need something specific that is not in there.
Note, the name gets overwritten with the name you have here.

Reply:
as per other comment, there is no reason to have `temperatureCelsius` if `temperatureC` exists. I'm going to reuse it.